### DISCUSSION TITLE
How to disable zoom option for ReactMapGL

### DISCUSSION THREAD

**Discussion by Jay-Bruh** [uber/react-map-gl#1548] [Q&A] [answered]
Currently I have my map component covering the full height and width of my page and I have other components below it. So whenever I scroll to go below, the map just zooms out and I'm not able to scroll down. So anyway I can disable the zooming option and freely scroll down to my other components without disturbing the map?

**Comment by Pessimistress** [comment, marked as answer]
[touchAction](https://visgl.github.io/react-map-gl/docs/api-reference/interactive-map#touchaction)

**Reply by Jay-Bruh** [reply]
Hey so I tried adding touchAction: "false" and "none" but that didnt change anything. I've attached my sandbox for reference https://codesandbox.io/s/full-popup-mapbox-stackoverflow-forked-gnqko?file=/src/App.js. Everytime I scroll my mouse, the map still zooms in and out

**Reply by Pessimistress** [reply]
Sorry, just read your question again. You want to set `scrollZoom` to `false`.

**Reply by Jay-Bruh** [reply]
This still didn't work out in my case. Can you review my sandbox and tell me where I'm going wrong? https://codesandbox.io/s/full-popup-mapbox-stackoverflow-forked-myu0i

**Reply by Pessimistress** [reply]
`scrollZoom` needs to be set as a ReactMapGL prop. Do not place it in `viewport` because the state is going to be overwritten.

Also, `false` and `"false"` are not the same thing.

**Reply by Jay-Bruh** [reply]
Ah yes, that did the trick thanks
